### ISSUE TITLE
Banner

### ISSUE THREAD

**Issue by thecodeduchess** [Hjekings/prj-rev-bwfs-dasmoto#1]
https://github.com/Hjekings/prj-rev-bwfs-dasmoto/blob/2e283c29014c57469fe9397f33decf11b657c3a2/Dasmoto's%20Arts%20%26%20Crafts/index.html#L13

Did you mean to omit the title background banner

<img width="1257" alt="screen shot 2018-05-08 at 8 06 32 pm" src="https://user-images.githubusercontent.com/8722966/39791134-97bb765e-52ff-11e8-8b9e-9066596784bb.png">

**Comment by Hjekings** [comment]
Yes, I omitted the title background banner because I couldn't figure out how to get it to display correctly.
